lib/dkim.js

```diff
@@ -463,8 +464,9 @@ function DKIMVerify(email, callback, sigs) {
 
         // The other DNS errors are operational in nature;
         sigs.result = sig.result = false;
-        sigs.issue_desc = sig.issue_desc = err;
-        return callback(err, sigs);
+        sigs.issue_desc = sig.issue_desc = err.message;
+
+        return callback(null, sigs); // returning the error AND the sigs breaks promisification... should do callback(null, sigs)
```

Comment:
right, it's returning the error that breaks promisification... that comment isn't totally clear :)

We don't have any logging in this lib, which is unfortunate. And we're exposing the error message as is, which might also be weird. The better fix might be to add msys-logger and log the error, and then return a hard-coded generic error message. What do you think?